generator/server_generator.go

```diff
@@ -367,12 +367,33 @@ func (gen *goServerGenerator) generateHandler(operation *Operation, parametersBu
 			}
 
 			for _, param := range parametersBucket.Query {
-				group := stmts.If(jen.Len(jen.Id("c").Dot("Request").Dot("URL").Dot("Query").Call().Index(jen.Lit(param.Name))).Op(">").Lit(0)).Block(
-					jen.If(jen.Id("err").Op(":=").Id("fromString").Call(jen.Id("c").Dot("Request").Dot("URL").Dot("Query").Call().Index(jen.Lit(param.Name)).Index(jen.Lit(0)), jen.Op("&").Id("request").Dot(strings.Title(identifier.MakeIdentifier(param.Name)))), jen.Id("err").Op("!=").Nil()).Block(
+				// ref: https://swagger.io/docs/specification/describing-parameters/#query-parameters
```

Comment:
The reference link should point to the OpenAPI 2.0 description for the query parameters (https://swagger.io/docs/specification/2-0/describing-parameters/#query-parameters)

Reply:
also fixed

Reply:
mmh wait! :)

Reply:
i fixed it